vitrina/templates/history.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block current_title %}{% translate "Veiksmų istorija" %}{% endblock %}
+
+{% block content %}
+
+	{% include 'component/tabs.html' %}
+
+    <div class="columns">
+        <div class="column is-three-quarters">
+            {% if history %}
+                <table class="table is-fullwidth">
+                    <tbody>
+                    <tr>
+                        <th>{% translate "Data/laikas" %}</th>
+                        <th>{% translate "Naudotojas" %}</th>
+                        <th>{% translate "Veiksmas" %}</th>
+                    </tr>
+                    {% for h in history %}
+                        <tr>
+                            <td>{{ h.date|date:"Y-m-d H:i:s" }}</td>
```

Comment:
```suggestion
                            <td>{{ h.date|date:"SHORT_DATE_FORMAT" }}</td>
```

Date format should be controlled by locale. People viewing this with different locale should see different format.

https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#date